### DISCUSSION TITLE
Min GCC version ?

### DISCUSSION THREAD

**Discussion by srinath-chakravarthy** [idaholab/moose#16560] [Q&A General] [answered]
Hi all, 
I noticed that there has been a change to the min gcc version. Does this mean that moose will not compile with gcc 4.8.5 ?? I have not tried and I am not sure if i should

**Comment by loganharbour** [comment, marked as answer]
The minimum supported gcc version has been bumped to 5.1.0. This does not necessarily mean that it will _not compile_ with 4.8.5... but that we don't officially support it.

**Reply by srinath-chakravarthy** [reply]
Will try and get back soon.

**Comment by abarun22** [comment]
I recall that i faced some issues compiling the newer version of MOOSE with
GCC 7.0 (in a recent build on CSD3). With lower versions, say, GCC 4.8.5,
it did build OK, but the test suite failed. May be it could be related to
the version of MPI loaded in the environment. But the good thing is that my
actual inelastic cases are pretty much up and running.

**Reply by loganharbour** [reply]
> I recall that i faced some issues compiling the newer version of MOOSE with GCC 7.0 

Can you share what issues you hit? Our default compiler stack is GCC 8.2.1.

**Reply by abarun22** [reply]
Thats a surprise. I did a rebuild with GCC 7.0 and it all seems fine now.

**Reply by milljm** [reply]
Our GCC default... it should be 9.3.0. Not sure where 8.2.1 is coming from.

**Reply by ngrilli** [reply]
On CentOS 7.8
I am following the manual installation described on the website
gcc 9.2.0
mpich 3.3
and it works fine
